### PR TITLE
update readme file

### PR DESCRIPTION
**Pull request by amit-mnnit**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```
Comment (sakshi-arora1) [inline LICENSE:5]: Assuming, we have got an approval for apache license.

File: packages/react-spectrum/README.md [hunk 2]
```diff
@@ -1,6 +1,111 @@
-# `@aemforms/af-react-spectrum`
+# React Components For Headless Adaptive Forms
+The document provides detailed information on React Components that you can use to render [Headless Adaptive Form](https://experienceleague.adobe.com/docs/experience-manager-headless-adaptive-forms/using/overview.html?lang=en). To learn about the capabilties of Headless Adaptive Forms, see [Overview of Headless Adaptive Forms](https://experienceleague.adobe.com/docs/experience-manager-headless-adaptive-forms/using/overview.html?lang=en)
```
Comment (sakshi-arora1) [inline packages/react-spectrum/README.md:2]: following spectrum

**Review by sakshi-arora1 — APPROVED** [review]
Minor comments.

**Review by sakshi-arora1 — APPROVED** [review]
Minor comments.

File: LICENSE (hunk 1 above)
Comment (amit-mnnit) [reply LICENSE]: Updated

File: packages/react-spectrum/README.md (hunk 2 above)
Comment (amit-mnnit) [reply packages/react-spectrum/README.md]: changed

**Review by nit23uec — APPROVED** [review]

**Pull request merged**